proposals/lip-0068.md

```diff
@@ -137,17 +137,17 @@ For a transaction `trs` to be valid, it must satisfy the following:
 * `trs.module` is an alphanumeric string, i.e., matches the regular expression `"^[a-zA-Z0-9]*$"`.
 * `trs.command` is an alphanumeric string, i.e., matches the regular expression `"^[a-zA-Z0-9]*$"`.
 
-These checks are performed in the [static validation stage][lip-0055#block-processing] of the block processing.
+The checks above are performed in the [static validation stage][lip-0055#block-processing] of the block processing.
+
+Further, it is the responsibility of every module to check in the "Command Verification" hook of a command that the bytes in `trs.params` satisfy the parameter schema of the respective command. 
 
 ### Serialization
 
 The serialization of an object of type `Transaction` is described in the following pseudocode.
 
 ```python
 def encodeTransaction(trs: Transaction) -> bytes:
```

Comment:
These functions now are pretty redundant, but still ok to keep them

Reply:
I would favor to keep them.